### ISSUE TITLE
hbar convention missing in the doc for evolution functions

### ISSUE THREAD

**Issue by ElieGouzien** [qutip/qutip#1360]
For now, there is no description of the chosen value of hbar in the documentation of the [evolution solvers](http://qutip.org/docs/latest/apidoc/functions.html#dynamics-and-time-evolution).

It seems that the convention `hbar = 1` is used, but it should be documented both in the doctrings of `sesolve`, `mesolve`… and in the [guide](http://qutip.org/docs/latest/guide/dynamics/dynamics-master.html).

I even think that the solved equation should be explicitly written in the docstring.

**Comment by nonhermitian** [comment]
Hbar is not needed, it is a constant that can be divided away. That is what QuTiP does. In the end your left with Hamiltonians expressed in units of angular frequency. The specific units of this angular frequency just set the units of time in the evolution and do not need to be entered explicitly.

**Comment by ElieGouzien** [comment]
I totally agree it is the right way to do numerics, but my point is that it should be written in the documentation.
For now the user have no way to answer the question "in which unit should I express my Hamiltonian", apart reading the source code.

It is especially misleading that the [guide](http://qutip.org/docs/latest/guide/dynamics/dynamics-master.html) write hbar in the equations but never say how it is handled.

**Comment by JRD801** [comment]
It is a wide spread habit among theoreticians to assume hbar =1 when dealing with quantum systems.

**Comment by ElieGouzien** [comment]
@JRD801 : not exactly, physicist don't "assume hbar = 1", but express their equations in a dimensionless way (said differently, they express the Hamiltonian in units of hbar).

Once again, I'm not questioning this choice (that I believe to be the best one); this issue point out that there is now way to be sure that is convention is followed by QuTiP (except by looking at the source code, what standard user never do).

By the way, it could be useful to explicitly write the equation solved in `sesolve` and `mesolve` docstrings.
